lib/config_selector.rb

```diff
@@ -40,6 +43,32 @@ def choose_players
     end
   end
 
+  def choose_markers(unordered_players)
+    markers = []
+
+    @console.marker_prompt(unordered_players[0])
+    markers << choose_marker(markers)
+
+    @console.marker_prompt(unordered_players[1])
+    markers << choose_marker(markers)
+
+    MarkerSelector.record(unordered_players, markers)
+  end
+
+  def choose_marker(selected_markers)
+    marker_choice = Input.choose
+
+    if MarkerSelector.wrong_length(marker_choice)
+      @console.output('Markers must be one single character. Please try again:')
+      choose_marker(selected_markers)
+    elsif MarkerSelector.duplicate(selected_markers, marker_choice)
```

Comment:
i think we should also consider a number an invalid marker since it could get really confusing seeing the numbers on the board.

Reply:
oof I didn't even think of that! will update, thanks

Reply:
updated